pom.xml

```diff
@@ -589,7 +589,8 @@
                 <sourceDirectory>${project.basedir}/src</sourceDirectory>
               </sourceDirectories>
               <excludes>
-                **/it/resources/**/*,**/test/resources/**/*,**/test/resources-noncompilable/**/*
+                **/it/resources/**/*,**/it/resources-noncompilable/**/*,
+                ,**/test/resources/**/*,**/test/resources-noncompilable/**/*
```

Comment:
excludes is string https://maven.apache.org/components/plugins/maven-checkstyle-plugin/check-mojo.html#excludes , so double comma is required to make exclude multi-line and valid.